app/main/containers/Search/index.js

```diff
@@ -109,7 +110,9 @@ class Search extends Component {
     // NOTE: when page refreshed (location.reload) componentWillUnmount is not called
     window.addEventListener('beforeunload', this.cleanup)
     this.electronWindow.on('show', this.focusMainInput)
+    this.electronWindow.on('show', this.updateElectronWindow)
     this.electronWindow.on('show', trackShowWindow)
+    this.electronWindow.on('move', this.moveElectronWindow)
```

Comment:
Since `moveElectronWindow` is not related to rendering I'd move it to `app/main/createWindow`. 

Another thing that we can do in `app/main/createWindow`. Now when you start application it looks like this: 

![](http://g.recordit.co/xai8KbDnZo.gif)

Would be cool to initialize `mainWindow` on correct position from config. What do you think, @maximbaz?